packages/components/src/components/page-error/page-error.tsx

```diff
@@ -1,15 +1,30 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isMobile } from '@deriv/shared';
 import DesktopWrapper from '../desktop-wrapper';
 import MobileWrapper from '../mobile-wrapper';
 import ButtonLink from '../button-link/button-link';
 import Text from '../text/text';
 
+type TMessageObject = { message: string; has_html?: boolean };
+
+type TPageErrorProps = {
+    buttonOnClick?: () => void;
+    buttonSize?: 'small' | 'medium' | 'large';
+    classNameImage?: string;
+    header: React.ReactNode;
+    image_url?: string;
+    messages: Array<TMessageObject | React.ReactNode>;
+    redirect_labels: string[];
+    redirect_urls?: string[];
```

Comment:
I might want to also update these 2 props in /packages/components/stories/page-error/stories/has-redirect.js where they are also passed as old `redirect_url` & `redirect_label` strings  🙂